docs/enable_disable_module.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Enable and disable a Kyma module
+---
+
+## Overview
+
+Your cluster comes with the Kyma custom resource (CR) already installed. It collects all metadata about the cluster, such as enabled modules, their statuses, or synchronization, using Lifecycle Manager. Lifecycle Manager uses `moduletemplate.yaml` to enable or disable modules on your cluster. 
+
+## Enable a module
+
+> **NOTE:** To quickly enable a module, go to your **Cluster Details** view and select **Add Module**. Then, choose the channel of your choice and click **Add** to choose and upload your module.
+
+> **TIP:** You can also configure your Kyma CR to enable a module manually. For more details, see [Kyma CR](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/technical-reference/api/kyma-cr.md).
```

Comment:
```suggestion
> **TIP:** You can also configure the Kyma CR to enable a module manually. For more details, see [Kyma CR](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/technical-reference/api/kyma-cr.md).
```
I would add the TIP at the end of the ## Enable a module section. What do you think?

Reply:
I second that, especially because it's ugly that we have two note/tip boxes right next to each other